core/src/main/java/hudson/scm/ChangeLogParser.java

```diff
@@ -52,6 +63,16 @@
         }
     }
 
+    /**
+     * Method for compatibility with AbstractBuild.
+     * SCM plugins should override this method to populate
+     * ChangeLogSet SCM attribute in AbstractBuild projects.
+     */
+    @Deprecated
+    public ChangeLogSet<? extends Entry> parseRun(AbstractBuild build, SCM scm, File changelogFile) throws IOException, SAXException {
```

Comment:
I would rather keep it as just "parse()" since it is not `Run`. Renaming both methods didn't help to avoid ambiguity I was referencing before